src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,32 @@
+package calculator;
+
+public class Calculator {
+
+    public static final String NUMBER_PATTERN = "[0-9]";
+    public static final String OPERATOR_PATTERN = "[+\\-*/]";
```

Comment:
```suggestion
    private static final String NUMBER_PATTERN = "[0-9]";
    private static final String OPERATOR_PATTERN = "[+\\-*/]";
```

내부 상수라면 `private` 으로 두는 것이 더 좋을 것 같아요 😄